velox/common/encode/Base64.h

```diff
@@ -77,14 +77,13 @@ class Base64 {
   static void encodeUrl(const char* data, size_t size, char* output);
 
   // compatible with www's Base64URL::encode/decode
-  // TODO rename encode_url/decode_url to encodeUrl/encodeUrl.
-  static std::string encode_url(const char* data, size_t len);
-  static std::string encode_url(const folly::IOBuf* data);
-  static std::string encode_url(folly::StringPiece text);
-  static void decode_url(
+  static std::string encodeUrl(const char* data, size_t len);
+  static std::string encodeUrl(const folly::IOBuf* data);
+  static std::string encodeUrl(folly::StringPiece text);
+  static void decodeUrl(
       const std::pair<const char*, int32_t>& payload,
       std::string& output);
-  static std::string decode_url(folly::StringPiece text);
+  static std::string decodeUrl(folly::StringPiece text);
 
   /*
    * Take str and writes the encoded string to out.
```

Comment:
Make base64_encode(decode)_string(..) naming consistent also since we are doing this exercise.

Reply:
Turns out that neither `base64_encode_string` nor `base64_decode_string` are implemented. The former has a commented implementation.
I suspect that this was an old implementation which was later replaced by implementing `decodeImpl` and `encodeImpl` and its various wrappers.

I can remove this completely.